src/_cffi_src/openssl/objects.py

```diff
@@ -24,11 +30,14 @@
 int OBJ_cmp(const ASN1_OBJECT *, const ASN1_OBJECT *);
 ASN1_OBJECT *OBJ_dup(const ASN1_OBJECT *);
 int OBJ_create(const char *, const char *, const char *);
+void OBJ_NAME_do_all(int, void (*) (const OBJ_NAME *, void *), void *);
 """
 
 MACROS = """
 /* OBJ_cleanup became a macro in 1.1.0 */
 void OBJ_cleanup(void);
+
+#define OBJ_NAME_TYPE_MD_METH ...
```

Comment:
We typically do this via "static const long" (assuming it is an integer) rather than letting cffi substitute on the define. If it's a char * then const char const *